library/src/main/java/de/agilecoders/wicket/webjars/collectors/AssetsMap.java

```diff
@@ -32,19 +32,29 @@
     private final SortedMap<String, String> fullPathIndex;
     private final AssetPathCollector[] collectors;
     private final String recentVersionPlaceHolder;
+    private static AssetsMap assetsMap;
```

Comment:
Why this needs to be static ?
As a general rule `static` should be avoided as much as possible in web applications because it leads to the whole application (WAR file) which may contain more than one Wicket application.
In addition this will make OSGi users very sad.
I'm afraid this change cannot be accepted.